### ISSUE TITLE
Menus don't work correctly when narrator is running

### ISSUE THREAD

**Issue by jaduff** [microsoft/vscode#116842]
With Narrator running, I press Alt+F to bring up the file menu. When I try to navigate down the File menu with the down arrow, it does not work. Navigating to adjacent menus with the left and right arrows also does not work. If I quit Narrator and try again, it works as expected.

**Comment by isidorn** [comment]
@jaduff thanks for letting us know. However we recommend using NVDA with VS Code since we test this out regularly.
For now I can leave this issue open, even though it sounds like a problem on the Narrator side.

**Comment by isidorn** [comment]
I actually prefer to close this as duplicate of https://github.com/microsoft/vscode/issues/10729
And move this issue into that one. Hope that makes sense.

**Comment by jaduff** [comment]
Thanks Isidor,

Will try NVDA.

**Comment by rbenson** [comment]
@jaduff - using narrator, try the right/left arrows vs up/down.

**Comment by isidorn** [comment]
Workaround for this is to turn off scan mode manually
